sagemaker_run_notebook/run_notebook.py

```diff
@@ -828,7 +833,12 @@ def proc(extras):
     return job
 
 
-RULE_PREFIX = "vp-sagemaker-setup-backdev-runNotebook-"
+def get_rule_prefix(session=None):
```

Comment:
Isn't it `get_lambda_name()` + `-`?

Reply:
You're right, I was way to scared to touch the 1000+ lines python file and went bit by bit. I kept the function but simplified it as you suggested.